CRM/Upgrade/Incremental/php/FiveSeventyTwo.php

```diff
@@ -29,6 +29,22 @@ class CRM_Upgrade_Incremental_php_FiveSeventyTwo extends CRM_Upgrade_Incremental
    */
   public function upgrade_5_72_alpha1($rev): void {
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+    $this->addTask('Remove localized suffixes from civicrm_mailing_group.entity_table', 'fixMailingGroupEntityTable');
+  }
+
+  /**
+   * Remove unwanted dbLocale suffixes from values in civicrm_mailing_group.entity_table.
+   *
+   * @see https://github.com/civicrm/civicrm-core/pull/29366
+   *
+   * @return bool
+   */
+  public static function fixMailingGroupEntityTable(): bool {
+    $updateQuery = 'UPDATE civicrm_mailing_group SET entity_table = "civicrm_mailing" WHERE entity_table LIKE "civicrm_mailing_%"';
```

Comment:
woohoo

Reply:
but - I don't see where we stop writing new instances of the locale name ?

Reply:
@eileenmcnaughton it stops here because these functions no longer return localized names:
https://github.com/civicrm/civicrm-core/blob/474535d3d44844e6a987822ee551adeb86bcc3a8/CRM/Mailing/BAO/Mailing.php#L2964-L2972